nixos-org/webserver.nix

```diff
@@ -154,6 +154,8 @@ in
                 #skins = [ ./wiki-skins ];
                 extraConfig =
                   ''
+                    $wgSiteNotice = "The NixOS wiki is outdated and is being shut down. If you wish to add or improve NixOS documentation, please grab a ticket from the [https://github.com/NixOS/nixpkgs/milestone/8 Move the wiki!] milestone.";
```

Comment:
Speaking as someone caught in the middle of this: I'd suggest pointing to an issue which clearly lays out why the wiki is moving, where it's moving to, how contributors can help out and then links to the milestone work queue.

Essentially: the message you've written here and the title of the milestone don't provide enough information for contributors to help out :).  It's confusing to read "move the wiki!" -- but then have no idea how to proceed.

Many of the issues in the milestone only link to a page to be moved.  Digging through issues to discover the process one should follow is frustrating.